.gitignore

```diff
@@ -17,3 +17,5 @@ src/Context/*/Infrastructure/Symfony/Bundle/DependencyInjection/Resources/*_para
 /phpunit.xml
 
 vendor/
+
+.idea
```

Comment:
This should be in your `.gitignore_global` -> https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

Why in the global and not here? Because it's not a project-related file, it's about a tool you use. .gitignore should contain only project-related files.